### DISCUSSION TITLE
How do i get the YOLO badge?

### DISCUSSION THREAD

**Discussion by DxstyLy** [community/community#19707] [Other Feature Feedback, Questions, & Ideas] [answered]
Help Please.

**Comment by sanrionpc** [comment, marked as answer]
Working way. You just need to:

1 Open your repository;
2 Create new branch;
3 Do some updates;
4 Then, create pull request to main branch;
5 IMPORTANT! Add reviewer. To do this, you need to add access to repository someone, then add him as reviewer;
6 After this, just click merge as you do it always;
7 Check your profile and see YOLO achievement.

**Reply by ayushgupta9198** [reply]
@VolcanoGunshot i have tried this but did not get the YOLO badge, let us know other ways to do the same
